cg/server/admin.py

```diff
@@ -17,6 +17,18 @@ def inaccessible_callback(self, name, **kwargs):
         return redirect(url_for('google.login', next=request.url))
 
 
+def view_family_sample_link(unused1, unused2, model, unused3):
```

Comment:
I guess the `unsused` ones are params that are passed by default?

Reply:
yes, I'm leaning towards another way of doing it, to use underscore (_, _, model, _), but then we can't delete them like we do now, what's your preference?

Reply:
The latter definitely is how I've seen everyone else dealing with unused and unneeded variables. So let's go for that

Reply:
It turns out that if you use the underscore more than once in the signature then it's considered to be a name collision...